Assets/Scripts/SimplePhysics.cs

```diff
@@ -153,7 +155,7 @@ private void Flip(float dir)
 
         private float GetXChange(float delta, float dir)
         {
-            for (int height = 0; height < 3; height++)
+            for (float height = 0.1f; height <= 1.9f; height+=0.9f)

```

Comment:
can't say for certain but it might be a bad idea to use floats for forloop guarding.
might be a better idea to use something like...
for (int i = 0; i < 3; ++i)
{
    var height = 0.1 + i * 0.9f;
}

This is guaranteed to always quit after 3 loops and doesn't require any assumptions about floating point precision (how precise is 0.9 in floating point and do we care??)

Take it with a grain of salt and give me your feedback